src/Core/Grid/Presenter/GridPresenter.php

```diff
@@ -81,7 +81,7 @@ public function present(GridInterface $grid)
         ];
 
         $this->hookDispatcher->dispatchWithParameters('action' . $definition->getId() . 'GridPresenterModifier', [
-            'presentedGrid' => $presentedGrid,
+            'presented_grid' => $presentedGrid,
```

Comment:
`$presentedGrid` is an array and thus cannot be modified by modules, is it intentional?

Reply:
yes, thinking about it, this is not a behavior we will change until the last major.

This is where people can change the formatting of their data, for instance.